lib/irb/context.rb

```diff
@@ -162,7 +162,7 @@ def initialize(irb, workspace = nil, input_method = nil)
     private_constant :KEYWORD_ALIASES
 
     def use_tracer=(val)
-      require_relative "ext/tracer"
+      require_relative "ext/tracer" if val
       @use_tracer = val
```

Comment:
I think this instance variable is never read.
Maybe we can set val to `IRB.conf[:USE_TRACER]` just like `context.use_loader=` does

Reply:
Yeah it's not used. But if we remove the setter, that's probably a breaking-change?
If you're fine with dropping it right now (since it has been broken anyway), we can just do

```rb
class Context
  def initialize(...)
    require_relative "ext/tracer" if IRB.conf[:USE_TRACER]
    # ...
  end
end
```

WDYT?

Reply:
I'm fine dropping, it

Or just remove the unused instance variable, this won't be a breaking change
```ruby
def use_tracer=(val)
  require_relative "ext/tracer" if val
end
```
Or change the ivar set to conf set. I think this is the original code's intention.
```ruby
def use_tracer=(val)
  require_relative "ext/tracer" if val
  IRB.conf[:USE_TRACER] = val
end
```

Reply:
Updated 👍